README.md

```diff
@@ -53,13 +53,13 @@ jobs:
 
 ## Inputs
 
-| Name        | Required | Description                                                                                                                            | Type   | Default |
-| ------------| :------: | ---------------------------------------------------------------------------------------------------------------------------------------| ------ | --------|
-| `token`     | ✓        | GitHub secret token, usually a `${{ secrets.GITHUB_TOKEN }}`                                                                           | string |         |
-| `toolchain` |          | Rust toolchain to use; override or system default toolchain will be used if omitted                                                    | string |         |
-| `args`      |          | Arguments for the `cargo clippy` command                                                                                               | string |         |
-| `use-cross` |          | Use [`cross`](https://github.com/rust-embedded/cross) instead of `cargo`                                                               | bool   | false   |
-| `name`      |          | Name of the created GitHub check. If running this action multiple times, each run must have a unique name.                             | string | clippy  |
+| Name                | Required | Description                                                                                                | Type   | Default |
+| ------------------- | :------: | ---------------------------------------------------------------------------------------------------------- | ------ | ------- |
+| `token`             |    ✓     | GitHub secret token, usually a `${{ secrets.GITHUB_TOKEN }}`                                               | string |         |
+| `toolchain`         |          | Rust toolchain to use; override or system default toolchain will be used if omitted                        | string |         |
+| `args`              |          | Arguments for the `cargo clippy` command                                                                   | string |         |
+| `use-cross`         |          | Use [`cross`](https://github.com/rust-embedded/cross) instead of `cargo`                                   | bool   | false   |
+| `working-directory` |          | Specify where rust directory is                                                                            | string | .       |
```

Comment:
I realize you just cherry picked an external commit, but can we fix this description so it's an actual sentence and is more descriptive of the actual use case?

Reply:
Merged, not cherry-picked.  But, sure, I can do that.